Slango/Slango/ButtonShadow.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ButtonShadow.swift
+//  Slango
+//
+//  Created by Fabiola S on 2/28/20.
+//  Copyright © 2020 Fabiola Saga. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ButtonShadow: UIButton {
```

Comment:
You might want to name this `ShadowedButton` to be more clear that this class is a button and not a shadow.

Reply:
Done!